app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -118,8 +123,13 @@
         reportType = org.acra.sender.HttpSender.Type.JSON,
         httpMethod = org.acra.sender.HttpSender.Method.PUT)
 public class CommCareApplication extends Application {
+
     private static final String TAG = CommCareApplication.class.getSimpleName();
 
+    // Tracking ids for Google Analytics
+    private static final String LIVE_TRACKING_ID = "UA-69708208-1";

```

Comment:
That would also work if we want local dev build to have the key, you would just need to add the key to your local gradle.properties